docs/sphinx/user_guide/config_options.rst

```diff
@@ -83,71 +111,90 @@ and their default settings:
 
 * **Examples, tests, warnings, etc.**
 
-     Variables that control whether RAJA tests, examples, or tutorial
-     exercises are built when RAJA is compiled:
-
-      ======================   ======================
-      Variable                 Default
-      ======================   ======================
-      RAJA_ENABLE_TESTS        On 
-      RAJA_ENABLE_EXAMPLES     On 
-      RAJA_ENABLE_EXERCISES    On 
-      RAJA_ENABLE_BENCHMARKS   Off
-      ======================   ======================
+     CMake variables can be used to control whether RAJA tests, examples, 
+     tutorial exercises, etc. are built when RAJA is compiled.
+
+.. note:: In the following tables, an option name followed by an asterisk ('*')
+          only exists in RAJA (i.e., no associated CMake or BLT variant). 
+          For the others (dependent options), it is recommended to use the
+          non-RAJA form (with ``RAJA_`` prefix removed) to enable features.
+
+      ========================   =========================================
+      Variable                   Default
+      ========================   =========================================
+      RAJA_ENABLE_TESTS          On 
```

Comment:
```suggestion
      (RAJA_)ENABLE_TESTS          On 
```

Do you think using parentheses for the dependent options might be clearer still? The most confusing thing (that we want to avoid) is that for something like CUDA, you _have_ to say ENABLE_CUDA=On (because RAJA_ENABLE_CUDA=On won't change the BLT option)

Reply:
That's a good idea. I will work on that change. Thanks.